modules/network-observability-topology.adoc

```diff
@@ -0,0 +1,35 @@
+= Using Topology
+Topology provides a graphical representation of the network flows and the amount of traffic. As an administrator, you can monitor the amount of traffic across the application by using the topology view.
+
+== Navigating to Topology
+As an administrator, you can navigate to topology view by using the following steps:
+
+. On the left navigation panel, click *Observe* → *Network Traffic*.
+. In the *Network Traffic* page, click the *Topology* tab.
+
+You can click each component to view the details and metrics of the component.
+
+== Configuring advanced options
+You can customize and export the view by using advanced options. To access the advanced options, click *Show advanced options* on the right-top corner.
+
+You can use the *Find in view* field to search the required components in the view. You can specify the grouping and display criteria by using the *Display options* dropdown. The options available are: 
+
+* *Layout*: To choose the layout of the graphical representation. The default value is *ColaNoForce*.
+
+* *Scope*: To choose the scope of components between which the network traffic flows. The default value is *Namespace*.
+
+* *Groups*: To enchance the understanding of ownership by grouping the components. The default value is *None*.
+
+* *Collapse groups*: To expand or collapse the groups. The groups are expanded by default. This option is disabled if *Groups* has value *None*. 
+
+* *Show*:To select the details that need to be displayed. All the options are checked by default.
+The options available are:
+
+** *Edges*: To show or hide the network traffic flow arrows.
+** *Edges label*: To show or hide the bytes/packets information on the edges. You cannot choose this option if the edges are not displayed. 
+** *Badges*: To show or hide the scope along with the label of the component. 
+
+* *Truncate labels*: Select the required width of the label from the drop-down list. The default value is *M*.
```

Comment:
Should we keep the same order here ? I feel it would help the user

![image](https://user-images.githubusercontent.com/91894519/205095532-5eacc19e-7b2f-47e2-acf0-f755a02ad259.png)

Reply:
Also we added tooltips on labels so feel free to reuse these descriptions to avoid repetitions:

![image](https://user-images.githubusercontent.com/91894519/205095933-6945cd71-2824-4b43-a252-19ebdff429db.png)
![image](https://user-images.githubusercontent.com/91894519/205095983-6115ba32-634f-4c0e-96ff-5adb356d3f7f.png)
![image](https://user-images.githubusercontent.com/91894519/205096019-f5a375b3-ae5d-44df-a999-124f8ba3345c.png)
![image](https://user-images.githubusercontent.com/91894519/205096079-0799d71c-9347-4f60-b800-f3d2b60dc5ef.png)
![image](https://user-images.githubusercontent.com/91894519/205096139-7dcfa28d-90b7-4897-8c8b-9ab9ebf89472.png)
![image](https://user-images.githubusercontent.com/91894519/205096215-7f5472bc-1245-4ca5-ad47-ad34505370f4.png)

Reply:
Yes, nice catch! will do that.

Reply:
Regarding reusing tooltip, I did go through those. However, there are some writing standards that need to be adhered to. So I have presented the information accordingly.

Reply:
Sure, thanks for the update !